tests/app/views/test_users.py

```diff
@@ -1393,6 +1408,25 @@ def test_400_response_if_framework_is_coming(self):
         response = self.client.get('/users/export/{}'.format(self.framework_slug))
         assert response.status_code == 400
 
+    def test_response_agreed_contract_variation(self):
+        self._setup()
+        self._put_complete_declaration()
+        self._post_complete_draft_service()
+        self._post_result(True)
+        self._post_framework_agreement()
+        self._set_framework_variation()
+        self._put_variation_agreement()
```

Comment:
This is adding a single test for the newly added field to the 400+ line `TestUsersExport` class that already exists to test this endpoint.  I have just followed the pattern that all the existing tests in this class already use.